tests/integration/test_integration.py

```diff
@@ -22,24 +22,119 @@ def test_getting_all_lists(admin_client, new_list):
     my_list = next(rl for rl in all_lists if rl.identifier == new_list.identifier)
 
 
-def test_get_list_items(admin_client, new_list_with_items):
-    record_list_items = admin_client.get_list_items(new_list_with_items)
+@pytest.mark.parametrize("resolve", [True, False])
+def test_get_list_items_one_unresolvable_item(
+    admin_client,
+    new_list_with_one_unresolvable_item,
+    resolve,
+):
+    if resolve:
+        record_list_items = admin_client.get_resolvable_list_items(
+            new_list_with_one_unresolvable_item
+        )
+    else:
+        record_list_items = admin_client.get_list_items(new_list_with_one_unresolvable_item)
+
+    assert isinstance(record_list_items, list)
+    assert all(isinstance(item, RecordListItem) for item in record_list_items)
+    assert len(record_list_items) == 0 if resolve else 1
```

Comment:
Minor: Given that the method call and expectation differs, I would think that splitting these collection of tests into separate tests might be clearer